ui/src/Components/NavBar.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import TypoGraphy from '@material-ui/core/Typography'
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar'
+
+
+
+function NavBar(props) {
```

Comment:
Use the below functional code format for all the components. Use 'const' in place of 'function'. 

import React from 'react';
import PropTypes from 'prop-types';

const NavBar = props => {
    return (
        <div>
            
        </div>
    );
};

NavBar.propTypes = {
    
};

export default NavBar;

Reply:
removing as not being used currently.